backend/trafficrules/src/main/java/com/coffeejug/trafficrules/service/UserService.java

```diff
@@ -29,7 +32,23 @@ public UserDto findById(String id) {
     }
 
     public UserDto save(UserDto userDto) {
-        User user = userRepository.save(modelMapper.map(userDto, User.class));
+
+        User user = null;
+        if (userDto.getUuid() != null) {
+            user = userRepository.findById(userDto.getUuid()).orElse(null);

```

Comment:
It's not recommended to use `orElse(null)`, it's much better to operate with `Optional` data structure